docs/src/pages/index.tsx

```diff
@@ -5,9 +5,53 @@
 
 import Layout from "@theme/Layout";
 import React from "react";
+import { v4 as uuidv4 } from "uuid";
 
+import appInsights from "@site/src/appInsights";
 import { Homepage } from "@site/src/components/home";
 
+/**
+ * Helper function to get a cookie value
+ * @param name - The name of the cookie
+ * @returns - The value of the cookie
+ */
+function getCookie(nam: string): string | undefined {
```

Comment:
```suggestion
function getCookie(name: string): string | undefined {
```

Curious how line 19 was working before, given this.

Reply:
huge catch--`name` is a global. Maybe this points to a way we should tweak website eslint config around globals?

Reply:
TIL! I didn't realize there was a `name` global!

Reply:
In the short term, might be worth renaming this variable to something a bit more specific. E.g. `cookieName`.

Reply:
It was also a recent typo in the PR history I believe, so might have been working as expected for a while.